src/android/RSA.java

```diff
@@ -64,11 +61,7 @@ public static void createKeyPair(Context ctx, String alias) throws Exception {
 						.setDigests(KeyProperties.DIGEST_SHA256, KeyProperties.DIGEST_SHA512)
 						.setEncryptionPaddings(KeyProperties.ENCRYPTION_PADDING_RSA_PKCS1)
 						.setRandomizedEncryptionRequired(true)
-						.setKeyValidityStart(notBefore.getTime())
-						.setKeyValidityEnd(notAfter.getTime());
-				if(Build.VERSION.SDK_INT >= Build.VERSION_CODES.N){
```

Comment:
Why was this deleted? The method only exists from that API forward IIRC.

Reply:
Already talked in slack but to register, we removed this if because we have another if before: if(Build.VERSION.SDK_INT > Build.VERSION_CODES.M)